src/managers/common/utils.ts

```diff
@@ -194,3 +197,41 @@ export async function getShellActivationCommands(binDir: string): Promise<{
         shellDeactivation,
     };
 }
+
+export async function notifyMissingManagerIfDefault(
+    managerId: string,
+    projectManager: PythonProjectManager,
+    api: PythonEnvironmentApi,
+) {
+    const defaultEnvManager = getDefaultEnvManagerSetting(projectManager);
```

Comment:
Does this notification get displayed just once? We should display this error only once.

I dont see any variable or the like that tracks whether this was already displayed once before.